pkg/reconciler/contour/config/contour.go

```diff
@@ -54,13 +58,22 @@ type visibilityValue struct {
 // NewContourFromConfigMap creates an Contour config from the supplied ConfigMap
 func NewContourFromConfigMap(configMap *corev1.ConfigMap) (*Contour, error) {
 	var tlsSecret *types.NamespacedName
+	var timeoutPolicyResponse string
+	var timeoutPolicyIdle string
 
 	if err := configmap.Parse(configMap.Data,
 		configmap.AsOptionalNamespacedName(defaultTLSSecretConfigKey, &tlsSecret),
 	); err != nil {
 		return nil, err
 	}
 
+	if err := configmap.Parse(configMap.Data,
+		configmap.AsString(timeoutPolicyResponseKey, &timeoutPolicyResponse),
+		configmap.AsString(timeoutPolicyIdleKey, &timeoutPolicyIdle),
```

Comment:
These can move to Parse invocation method above

Reply:
Done